src/core/index.ts

```diff
@@ -0,0 +1,358 @@
+import axios from 'axios'
+import { BigNumber } from 'bignumber.js'
+import bodyParser from 'body-parser'
+import debug from 'debug'
+import express from 'express'
+import uuid from 'uuid/v4'
+import { createController } from './controllers'
+import { isSafeKey, SettlementStore } from './store'
+import { fromQuantity, isQuantity, isValidAmount } from './utils/quantity'
+import { retryRequest } from './utils/retry'
+
+/**
+ * Settlement system -specific functionality that each settlement engine
```

Comment:
not sure if the space between the system and specific is meant to be there or the `-` should not be there